book.toml

```diff
@@ -3,7 +3,7 @@ authors = ["Willem Wyndham"]
 language = "en"
 multilingual = false
 src = "src"
-title = "RAEN|ИƎAЯ"
+title = "RAEN Guide"
```

Comment:
also yes I got rid of the backwards name. Seems too proud of itself, and sorta confusing. This is where the brand name should go; the thing that shows up in the tab title. "RAEN Guide" seems stronger to me. Who really cares that "RAEN" is "NEAR" backwards. Shouting about it is not the purpose of this guide.

Reply:
I do think an eventual logo could subtly point it out.

Until then, maybe we can just state it in the opening line. 

> ## Welcome to RAEN 🌧
>
> RAEN (NEAR spelled backward) is a build tool for NEAR.

Reply:
Or we could have an “about the name” section, [like AVA](https://github.com/avajs/ava#how-is-the-name-written-and-pronounced)?

> ## About the name
>
> “RAEN” is “NEAR” spelled backward. It is pronounced the same as “rain”.

Reply:
Yeah that sounds good.  I like the latter on the readme.

Reply:
Done!